kale/sqs.py

```diff
@@ -41,12 +42,12 @@ def __init__(self, *args, **kwargs):
             if settings.MESSAGE_QUEUE_USE_PROXY:
                 # Used only in Dev environment which uses ElasticMQ instead of
                 # SQS. Hence it uses a different boto api to connect to a proxy
-                conn = boto.sqs.connection
-                self._connections[conn_str] = conn.SQSConnection(
-                    proxy_port=settings.MESSAGE_QUEUE_PROXY_PORT,
-                    proxy=settings.MESSAGE_QUEUE_PROXY_HOST, is_secure=False,
-                    aws_access_key_id=aws_access_key_id,
-                    aws_secret_access_key=aws_secret_access_key)
+                region = boto.sqs.regioninfo.RegionInfo(name='elasticmq', endpoint=settings.MESSAGE_QUEUE_PROXY_PORT)
+                self._connections[conn_str] = boto.connect_sqs(aws_access_key_id=aws_access_key_id,
+                    aws_secret_access_key=aws_secret_access_key,
+                    region=region,
+                    is_secure=False,
+                    port=settings.MESSAGE_QUEUE_PROXY_PORT)
```

Comment:
i don't have enough context to approve or reject this.  i've mucked with `RegionInfo` before, but i don't recall the details.  eg why is the name `elasticmq`?

Reply:
also, does this break backward compat?  i know there are non-nextdoorians that use this.

Reply:
elastimq is an arbitrary value.  I changed it to proxy as suggested by @swaggy.  Besides running nosetests, how can I test out backwards compatibility?

Reply:
I did a backwards compat test by pointing to this version of kale, and using elasticmq container created by the docker-compose in nextdoor.com:

```(nextdoor3) ✔ 11:11 ~/src/nextdoor.com [master|✚ 3…1⚑ 4] $ ./apps/manage.py populate_data --dataset=default --publish --country=US
nextdoor.util.alert DEBUG: Initializing Alerter object for the first time...
nextdoor.dev_datasets.management.commands.populate_data INFO: Running populate_data command
nextdoor.tasks.nd_task.populatedatatask DEBUG: Received task: nextdoor.dev_datasets.management.commands.populate_data.PopulateDataTask

settings.MESSAGE_QUEUE_USE_PROXY True
settings.MESSAGE_QUEUE_USE_PROXY exists
MESSAGE_QUEUE_PROXY_HOST 0.0.0.0
MESSAGE_QUEUE_PROXY_PORT 9324

nextdoor.tasks.nd_task.populatedatatask DEBUG: Published task.
```

Here is the output with this version of kale, and using the elasticmq container created by the Kubernetes changes:

```(nextdoor3) ✔ 11:40 ~/src/nextdoor.com [ambassador|●1✚ 16…1⚑ 3] $ ./apps/manage.py populate_data --dataset=default --publish --country=US
nextdoor.util.alert DEBUG: Initializing Alerter object for the first time...
nextdoor.dev_datasets.management.commands.populate_data INFO: Running populate_data command
nextdoor.tasks.nd_task.populatedatatask DEBUG: Received task: nextdoor.dev_datasets.management.commands.populate_data.PopulateDataTask

settings.MESSAGE_QUEUE_USE_PROXY True
settings.MESSAGE_QUEUE_USE_PROXY exists
MESSAGE_QUEUE_PROXY_HOST elasticmq.localhost.com
MESSAGE_QUEUE_PROXY_PORT 80

nextdoor.tasks.nd_task.populatedatatask DEBUG: Published task.```